pkg/tecdsa/signing/gen/pb/message.proto

```diff
@@ -50,4 +50,10 @@ message TSSRoundSevenMessage {
     uint32 senderID = 1;
     bytes payload = 2;
     string sessionID = 3;
+}
+
+message TSSRoundEightMessage {
+    uint32 senderID = 1;
+    bytes payload = 2;
```

Comment:
`broadcastPayload` please 🙏🏻 as noted in the review of previous PRs in this series.

Reply:
Addressed by https://github.com/keep-network/keep-core/pull/3271